### ISSUE TITLE
Anti-drop implant: Item can get stuck in hand

### ISSUE THREAD

**Issue by Ralta** [ParadiseSS13/Paradise#8178]
**Problem Description**:
When using the anti-drop implant, occasionally bugs out and prevents you from dropping an item in-hand. If you force the item out of that hand (e,g. by amputating the hand), any other person who picks up the item won't be able to drop it either (regardless of if they have an implant).

**What did you expect to happen**: 
To be able to drop the item, and for a non-implanted person to also be able to drop the item.

**What happened instead**:
Unable to drop the item; had to amputate the hand the first time, then ahelp the second time.

**Why is this bad/What are the consequences:**
Inability to drop item is bad enough for the person affected, but the item can then go on to affect someone else too.

**Steps to reproduce the problem**:
Implant person with Anti-Drop Implant.
Bug occurs that prevents item from being dropped.
Force the item to drop by amputating hand.
Pick up the item as un-implanted person.

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was
anything else out of the ordinary happening?)**: 
Playing as Brig Physician, not implanted. Warden had been implanted about 30 mins prior.

Similar to, but not quite a duplicate of #7300

**Comment by tigercat2000** [comment]
`When using the anti-drop implant, occasionally bugs out and prevents you from dropping an item in-hand`
... isn't that the entire point of the anti-drop implant?

**Comment by Ralta** [comment]
You should still be able to put it back in your bag/belt/pockets etc. Otherwise you would literally only be able to hold two items for the remainder of the shift.

Apologies, I should have been clearer when I said "drop".

**Comment by TullyBurnalot** [comment]
@tigercat2000 I believe there's an open issue for this somewhere, I'll have a look.

**Comment by TullyBurnalot** [comment]
@Ralta https://github.com/ParadiseSS13/Paradise/issues/7300 ?

**Comment by Ralta** [comment]
@TullyBurnalot Yup, and that had a fix associated with it which should have resolved this too I'm theory. I referenced it in my original post :-)

**Comment by Ralta** [comment]
*in
Damned autocorrect.

**Comment by IK3I** [comment]
that pr got closed because maintainers wanted a full rewrite of hand code instead of the hacky make it work version. And rewriting hand code is a nightmare and a half.

**Comment by Ralta** [comment]
@crazylemon64 I think you've missed the point; the item itself becomes 'locked' in a state of being undroppable. If the person that's implanted activates the no-drop and then is forced to drop it by means of amputation, anyone else who picks up the item (regardless of whether they have the implant or not) will be unable to drop it.

To give an example: 
```Player 1 has no-drop. They have a wrench in-hand and activate no-drop. 
Player 2 cuts Player 1's hand off with an e-sword. 
Player 2 picks up the now dropped wrench. They are no longer able to drop it. ```
 
I haven't tried this since the initial post, but I assume it's still an issue. A niche one, but still an issue.
